### DISCUSSION TITLE
Ironing infill is greyed out

### DISCUSSION THREAD

**Discussion by AndersDoverud** [supermerill/SuperSlicer#2555] [Q&A] [answered]
I want to iron a infill, but can't change it, how do I do it?
[infill](https://user-images.githubusercontent.com/80225736/159334217-19d03f21-014e-483f-b8a1-cdcbeaf607fe.jpg)

**Comment by NoTaMu** [comment]
You have to set the solid pattern to ironing ;)

**Reply by AndersDoverud** [reply]
Do you mean the "Ironing post-process" under?
That didn't work...

**Reply by NoTaMu** [reply]
I mean the solid dropdown list here:
![Sin título](https://user-images.githubusercontent.com/38410923/159340970-c58d6aac-19b1-42ea-8b78-2bb003f7c51d.jpg)

**Reply by AndersDoverud** [reply]
Thank you!

**Comment by neophyl** [comment, marked as answer]
There are 2 ironing methods in Super Slicer.  There is the older method that pre dates the version added by Prusa in PS and brought in via a merge.  The one circled is for adjusting the Prusa method, which needs to be selected in the dropdowns as NoTaMu has pointed out.

The older method is the Ironing Post Process (the section below the area circled).  This you need to enable with the tickbox called Enable ironing post process.  It will work with any of the top fill patterns but its method of application is different. Both methods are separate so you could in theory iron twice is you set it up that way (I wouldn't recommend doing that).  The older Super Slicer method has the 'slight' advantage that it will also iron perimeters too and not just the infill pattern.

**Reply by AndersDoverud** [reply]
Thank for a good explanation. :)

**Reply by supermerill** [reply]
Just a note: the post-process is the one made by prusa. I only did the pattern.

**Reply by neophyl** [reply]
Sorry if I got them the wrong way around.

**Reply by supermerill** [reply]
np
